templates/proxy.php.erb

```diff
@@ -0,0 +1,9 @@
+<?php
+  $allowed_urls = [
+<% @css_sheets.each do |sheet| -%>
+  <%= sheet['local']?'':'"'+sheet['filename']+'",' %>
+<% end -%>
+  ];
+
+  if( in_array( $_GET[ 'url' ], $allowed_urls ) )
```

Comment:
Some minor comments. `if (isset($_GET['url']) && is_string($_GET['url']) && in_array(...` would be more correct as it would not generate a notice if `url` is not provided or an error if `$_GET['url']` is an array.